problems/helpers/list.py

```diff
@@ -0,0 +1,23 @@
+from core.list import ListNode
```

Comment:
I don't like this file naming, also. viewers/printers?)
I love your idea to organize our extracode in structure)

Reply:
But this helper is only for the ListNode.